### ISSUE TITLE
Automatic decoding in code blocks

### ISSUE THREAD

**Issue by evi1ox** [marktext/marktext#1856]
```
<iframe src=javas&#x09;cript:alert(1)></iframe> //Tab
<iframe src=javas&#x0A;cript:alert(1)></iframe> //回车
<iframe src=javas&#x0D;cript:alert(1)></iframe> //换行
<iframe src=javascript&#x003a;alert(1)></iframe> //编码冒号
```

`&#x09;  &#x0A;  &#x0D; &#x003a;`It decodes itself.

**Comment by evi1ox** [comment]
<img width="512" alt="xxxx" src="https://user-images.githubusercontent.com/24634543/72079477-f2eb1f80-3335-11ea-9504-fab6ebe09312.png">

**Comment by evi1ox** [comment]
Copying the code block part of the code will html escape
```php
&lt;?php $content = &#39;&lt;?php exit; ?&gt;&#39;;
```

**Comment by fxha** [comment]
Fixed in v0.17.0
